config/locales/en.yml

```diff
@@ -53,8 +53,8 @@ en:
       teacher_reference_number: "What's your teacher reference number?"
       national_insurance_number: "What's your National Insurance number?"
       student_loan_amount:
-        "How much student loan did you repay while you were at %{claim_school_name} between 6 April XXXX and 5 April
-        XXXX?"
+        "Exactly how much student loan did you repay while you worked at Penistone Grammar School between 6 April XXXX
```

Comment:
Should be `Exactly how much student loan did you repay while you worked at %{claim_school_name}`

Reply:
Changing now.